linux_kernel_modules/rtc-pcf85063/rtc-pcf85063.c

```diff
@@ -0,0 +1,233 @@
+/*

```

Comment:
- This file is in DOS format (CRLF) line endings.  It should be in UNIX format.
- You should include a README file which describes where the code came from.  So for example you could say that the driver was copied out of the Linux kernel source from drivers/rtc/rtc-pcf85063.c at tag v4.18.3 (or whatever the right version is)
- File should be utf-8 encoded.  It is encoded this way in the kernel source.